### DISCUSSION TITLE
Stanford CS231n: CNN for Visual Recognition - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#173] [Announcements]
# Stanford CS231n: CNN for Visual Recognition - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%B7%B1%E5%BA%A6%E5%AD%A6%E4%B9%A0/CS231/

**Comment by RZII** [comment]
请问作业的Autograder是怎么解决的呢？我看官网有colab是colab会有feedback吗？

**Reply by ccmoony** [reply]
jupyter notebook里有检查答案正误的代码

**Comment by ccmoony** [comment]
这是我的assignments代码实现https://github.com/ccmoony/cs231n_assignment.git
欢迎大家提issue与我交流！

**Reply by 1nram1** [reply]
大佬，请问这门课的作业，就看colab的输出结果，就能检查答案正误了吗？其实不是很需要autograder?谢谢

**Reply by ccmoony** [reply]
是这样的

**Comment by Eug-Liu** [comment]
有本地测试吗请问

**Comment by rainy-xie** [comment]
找人一起写cs231n 2024，有组队的么

**Reply by 2090741942** [reply]
一起啊朋友！
email: 2090741942@qq.com

**Reply by Tisome** [reply]
我也是刚开始学，一起？
email: 1244172541@qq.com

**Reply by DDDIGHE** [reply]
还来得及吗，2522149073@qq.com

**Reply by JertaHe** [reply]
有群没？一起
2804769591@qq.com

**Reply by QingMoQian** [reply]
还来得及吗，3103137737@qq.com

**Reply by xiahy23** [reply]
有群吗，我也准备开始！
email: 1825607189@mails.qq.com

**Reply by JaxnBai** [reply]
有群吗，我也准备开始！
email: 2226352665@qq.com

**Reply by 114514DJJZ** [reply]
请问现在还有吗？3549415476@qq.com   感谢大佬

**Comment by QingMoQian** [comment]
这是我的2024-assignments实现:
https://github.com/QingMoQian/CS231n-2024-Assignments
欢迎指正

**Reply by QingMoQian** [reply]
也简单说了一下学习心得

**Comment by stibiums** [comment]
如何使用colab完成作业呢

**Reply by butforyou** [reply]
官网有教程可以看看指导

**Comment by HankDai13** [comment]
CS231n 的 2025 Spring版本的视频已经开源。https://www.youtube.com/playlist?list=PLoROMvodv4rOmsNzYBMe0gJY2XS8AQg16

**Comment by helenabieber28-max** [comment]
在跑assignment3中transformer代码的时候记得把colab文件的运行时类型版本号改成官网发布作业文件的时间点，不然可能因为版本不同随机数产生的数据不同测试一直不通过

**Reply by AniMikasa** [reply]
是的

**Reply by ShuaiZhou302** [reply]
感谢！
